test/shared/src/main/scala/zio/test/Gen.scala

```diff
@@ -181,6 +181,24 @@ object Gen extends GenZIO with FunctionVariants with TimeVariants {
   val alphaNumericChar: Gen[Random, Char] =
     weighted(char(48, 57) -> 10, char(65, 90) -> 26, char(97, 122) -> 26)
 
+  /**
+   * A generator of alpha characters.
+   */
+  val alphaChar: Gen[Random, Char] =
```

Comment:
Can we alphabetize please? Otherwise looks great!

Reply:
Sorry, didn't get what you mean.
Alphabetize random char generator?

Reply:
Oh sorry. Just put the constructors in alphabetical order with all the existing constructors. So like right now `whiteSpaceChars` is near the top of the file above the existing `alphaNumericString` generator but it should be towards the end of the file after the `weighted` constructor. It doesn't actually change the logic of your implementation but just makes it easier for users and contributors to find things in the file, especially when it is a large one like this one.

Reply:
Ok, I think now they are alphabetized:)